.php_cs.dist

```diff
@@ -17,6 +17,7 @@ return PhpCsFixer\Config::create()
         'combine_nested_dirname' => true,
         'list_syntax' => ['syntax' => 'short'],
         'visibility_required' => ['property', 'method', 'const'],
+        'ternary_to_null_coalescing' => true,
```

Comment:
This should then be added to the @Symfony ruleset of pho-cs-fixer, too

Reply:
folks, do not hesitate to raise PR for Fixer repo ;)